fcl/dunefd/caf/cafmaker_dune10kt_1x2x6_runreco-nuenergy.fcl

```diff
@@ -0,0 +1,66 @@
+#include "energyreco.fcl"
+#include "angularreco.fcl"
+#include "standard_cafmaker_dune10kt_1x2x6.fcl"
+
+dunefd_hd_producers:
+{
+    #neutrino energy reco
+    energyrecnumu:      @local::dunefd_nuenergyreco_pandora_numu
+    energyrecnumurange: @local::dunefd_nuenergyreco_pandora_numu_range
+    energyrecnumumcs:   @local::dunefd_nuenergyreco_pandora_numu_mcs
+    energyrecnue:       @local::dunefd_nuenergyreco_pandora_nue
+    energyrecnc:        @local::dunefd_nuenergyreco_pandora_nc
+    #angle reco configuration
+    anglereconue:       @local::dunefd_nuangularreco_pandora_nue
+    anglereconumu:      @local::dunefd_nuangularreco_pandora_numu
+    anglereconuepfps:   @local::dunefd_nuangularreco_pandora_nue_allpfps
+    anglereconumupfps:  @local::dunefd_nuangularreco_pandora_numu_allpfps
+}
+
+dunefd_hd_ereco:
+[
+    energyrecnumu,
+    energyrecnumurange,
+    energyrecnumumcs,
+    energyrecnue,
+    energyrecnc
+]
+
+dunefd_hd_anglereco:
+[
+    anglereconue,
+    anglereconumu,
+    anglereconuepfps,
+    anglereconumupfps
+]
+
+physics.producers: {
+    @table::dunefd_hd_producers
+}
+
+physics.prod: [
+    @sequence::dunefd_hd_ereco
+]
+
+physics.trigger_paths: [ prod ] 
+
+
+physics.producers.energyrecnumu.HitLabel: "hitfd"
+physics.producers.energyrecnumu.WireLabel: "tpcrawdecoder:gauss"
+physics.producers.energyrecnumu.HitToSpacePointLabel: "hitfd"
+
+physics.producers.energyrecnumurange.HitLabel: "hitfd"
+physics.producers.energyrecnumurange.WireLabel: "tpcrawdecoder:gauss"
+physics.producers.energyrecnumurange.HitToSpacePointLabel: "hitfd"
+
+physics.producers.energyrecnumumcs.HitLabel: "hitfd"
+physics.producers.energyrecnumumcs.WireLabel: "tpcrawdecoder:gauss"
+physics.producers.energyrecnumumcs.HitToSpacePointLabel: "hitfd"
+
+physics.producers.energyrecnue.HitLabel: "hitfd"
+physics.producers.energyrecnue.WireLabel: "tpcrawdecoder:gauss"
+physics.producers.energyrecnue.HitToSpacePointLabel: "hitfd"
+
+physics.producers.energyrecnc.HitLabel: "hitfd"
+physics.producers.energyrecnc.WireLabel: "tpcrawdecoder:gauss"
+physics.producers.energyrecnc.HitToSpacePointLabel: "hitfd"
```

Comment:
Do these need setting here?  The upstream fcl blocks e.g. `dunefd_nuenergyreco_pandora_numu` don't already set the variables correctly?

Reply:
Apparently no, they are set differently in energyreco.fcl -> https://github.com/DUNE/dunereco/blob/fb5a5d2acef112785e3078c8d23e2c617114518c/dunereco/FDSensOpt/energyreco.fcl#L34

I admit this might be better to modify it directly there, I can take care of this.